### PR TITLE
MAT-4948 measure group population id required

### PR DESCRIPTION
**Pull request by sb-cecilialiu**
## MADiE PR Template

Jira Ticket: [MAT-4948](https://jira.cms.gov/browse/MAT-4948)
(Optional) Related Tickets:

### All Submissions
* [x] This PR has the JIRA linked.
* [ ] Required tests are included.
* [x] No extemporaneous files are included (i.e. Complied files or testing results).
* [x] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [x] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: src/main/java/gov/cms/madie/models/measure/Group.java [hunk 1]
```diff
@@ -21,6 +22,7 @@
 @AllArgsConstructor
 @ValidGroupScoringPopulation
 @ValidMeasureObservation
+@ValidGroupPopulationId
```
Comment (adongare) [inline src/main/java/gov/cms/madie/models/measure/Group.java:25]: can we not use @NotBlank annotations from javax.validations?

Comment (sb-cecilialiu) [reply src/main/java/gov/cms/madie/models/measure/Group.java:25]: @adongare That was my first thought as well, but using @NotBlank with @Id on Population.id field didn't work.

Comment (adongare) [reply src/main/java/gov/cms/madie/models/measure/Group.java:25]: I'm thinking this has something to do with @Valid vs  @Validated.

**Review by adongare — DISMISSED** [review]

File: src/main/java/gov/cms/madie/models/measure/Group.java (hunk 1 above)
Comment (sb-cecilialiu) [reply src/main/java/gov/cms/madie/models/measure/Group.java]: @adongare Thanks so much for your help Ashok! I reverted the use of custom validation. Please have a look again. Thanks!

**Review by adongare — APPROVED** [review]
looks good!

**Pull request merged**